source/Halibut/HalibutRuntimeBuilder.cs

```diff
@@ -21,7 +22,19 @@ public class HalibutRuntimeBuilder
         AsyncHalibutFeature asyncHalibutFeature = AsyncHalibutFeature.Disabled;
         Func<string, string, UnauthorizedClientConnectResponse> onUnauthorizedClientConnect;
         HalibutTimeoutsAndLimits halibutTimeoutsAndLimits;
-
+        IBaseStreamWrapper baseStreamWrapper = new NoOpBaseStreamWrapper();
```

Comment:
I'm not too fond of this. Can this not be null if it's not set and when null it does not change the call chains